ups-connector-demo/pom.xml

```diff
@@ -2,13 +2,13 @@
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>ups.connector.demo</groupId>
+  <groupId>com.axonivy.connector.ups</groupId>
   <artifactId>ups-connector-demo</artifactId>
   <version>1.0.0-SNAPSHOT</version>
```

Comment:
please make it consistent with the version you defined in pom of ups-connector-product (10.0.14-SNAPSHOT)